### ISSUE TITLE
Measures are returned with null values

### ISSUE THREAD

**Issue by brettgriffin** [fleet-data/fleet-source-itunes-connect#1]
Version 0.0.51 returns null values for most measures:

```
{
	"date": "2016-12-26 00:00:00.0",
	"installs": null,
	"sessions": null,
	"pageViewCount": null,
	"rollingActiveDevices": null,
	"crashes": null,
	"payingUsers": null,
	"units": null,
	"sales": null,
	"iap": null,
	"impressionsTotal": 11.0,
	"impressionsTotalUnique": null,
	"pageViewUnique": null,
	"domainReferrer": null,
	"appVersion": null,
	"campaignId": null,
	"platform": "iPad",
	"platformVersion": null,
	"region": null,
	"storefront": null
}
```

**Comment by jre247** [comment]
fleetdata/fleet:0.0.73 fixes this issue.

**Comment by brettgriffin** [comment]
Resolved in Fleet `0.0.73`
